public/stylesheet.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+    font-family: 'header-font';
+    src: url('Chomsky.otf');
+}
+
+@font-face {
+    font-family: 'general-font';
+    src: url('LinLibertine_R.ttf');
+}
+
+* {
+  box-sizing: border-box;
+  font-family: 'general-font';
+}
+
+body {
+  width: 1280px;
+  margin: auto;
+}
+
+/* Header */
+
+.header-bar {
+  display: grid;
+  grid-template-rows: auto / auto
+}
+
+.header {
+  font-family: 'header-font';
+  font-size: 50pt;
+  text-align: center;
+}
+
+.date-time {
+  text-align: center;
+}
+
+/* Article */
+
+.article-container {
+  display: grid;
+  grid-template: 15em auto / min-content auto;
+  align-items: center;
+  color: black;
+  margin: 2em;
+  border: 1px solid black;
+}
+
+.article-image {
+  height: 100%;
+}
+
+.article-text-container {
+  display: grid;
+  grid-template: auto auto auto / 10em auto;
+  padding: 2em;
+}
+
+.article-headline {
+  grid-column: 1 / 3;
+  align-self: center;
+  font-size: 200%;
+  padding-bottom: 0.25em;
+}
+
+.article-section {
+  align-self: left;
+  text-transform: capitalize;
+}
+
+.article-snippet {
+  padding-top: 1.5em;
+  grid-column: 1 / 3;
+}
+
+/* Footer */
```

Comment:
cloned this and looks really neat and tidy - obvious already you're a dab hand at css. Looks really nice with the description under each header - the spec asked for the description to appear once clicked on but I think how yours is set up works really well. Maybe only the first couple sentences could be revealed at first? And maybe some sort of colour change or underline of thetitle so when you hover over it you definitely know it's going to take you to an external link